java/code/src/com/suse/manager/webui/websocket/RemoteMinionCommands.java

```diff
@@ -72,6 +76,7 @@
 
     private CompletableFuture failAfter;
     private List<String> previewedMinions;
+    private static ExecutorService eventHistoryExecutor = Executors.newFixedThreadPool(10);
```

Comment:
How was the number '10' chosen for the pool? If it is just an arbitrary choice, I'd prefer to use `newCachedThreadPool` here instead.

Reply:
It's arbitrary. I'll change it to `newCachedThreadPool` and  increase it to `20` to be on par with:
https://github.com/uyuni-project/uyuni/blob/5f5560129b6c4b94a84297eb492083e4d837ddce/java/code/src/com/suse/manager/webui/services/impl/SaltService.java#L176-L179

Reply:
Ups, I've overlooked, the `newCachedThreadPool` doesn't have a limit.